### DISCUSSION TITLE
GitHub Community: Here's to an incredible 2023 🎆

### DISCUSSION THREAD

**Discussion by grandmas-favorite** [community/community#84131] [Announcements]
2023 was an epic year for our Community, boasting nearly **double** the volume and total solutions authored as last year...all thanks to you! You make it possible for us to continue building around the launch of new & innovative products and improved user experience & accessibility!

Take a minute to be proud of your contribution and celebrate the start of the new year by checking out some highlights from 2023, and if you want, take an opportunity to share gratitude with members who had a positive impact on you in the Community. Add a comment below to say thank you! 🥳 

<img src="https://github.com/community/community/assets/144172738/2b8d9561-2ea0-47c7-83cf-41d7cfc0a83f" alt="image" width="50%" height="auto">

### Long Description:
The infographic provides a snapshot of different data points from the GitHub Community over the year 2023. It has a purple, abstract background that simulates the shapes of a lava lamp. 

There are five different graphics to correspond with five different data points and descriptions. The graphics and descriptions are as follows: 

A cartoon graphic of a person typing on their laptop that is pink, purple and beige, next to a header entitled “23,546 Total Discussions”, and the text "The Community was extra busy this year, almost doubling the volume of discussions we saw in 2022!".

Below, a graphic of several people putting together puzzle pieces that form a lightbulb that is beige, purple, and blue with the header: "9,824 Solutions Offered", and the text: "Our members continued to provide the value of community by dishing out over 9,000 answers throughout the year!"

Then, an orange, purple and blue graphic of three people exchanging messages in a circle is paired with the header "Most Active
Categories", followed by the list: 1. Actions - 3,682 posts, 2. Education - 2,583 posts and 3. Copilot - 2,150 posts

Next, an orange and pink graphic of a message box with a heart inside is next to the header: "Biggest Month" and the text: "The Community was on fire in August, creating a total of 2,253 posts. The average posts/per month for 2023 was 1,962."

Lastly, there is multicolored graphic of 5 people joined by a thread with the header: "Most Active Members" and text: "These members went above and beyond to be outstanding community citizens: @jge162, @airtower-luna, @jsoref, @GhostOf0days, @linuxlawson

**Comment by priyankarpal** [comment]
great to see 👍🏻

**Reply by GF-Elite** [reply]
Hmmm

**Reply by Kiranskkiru14** [reply]
Super

**Reply by butterfly0625** [reply]
👍

**Reply by RohRakun77** [reply]
🥳

**Reply by gemy6315** [reply]
[]()@[]()

**Reply by LaurajStarks** [reply]
🤔

**Comment by RohailJohn** [comment]
Good to see 🙂

**Reply by samzafar26** [reply]
Hello there

**Reply by RohailJohn** [reply]
hi

**Comment by rahimbhuiyan2030** [comment]
Dear Sir  how can is this apps possible new  register MoOver-pro

**Comment by Habibsultani** [comment]
Greate

**Reply by rahimbhuiyan2030** [reply]
How can registration option do open the apps please I asking your below 

![Screenshot_20231216_194740_One UI Home](https://github.com/community/community/assets/108199742/073ec974-959c-4878-a669-57dfb29dd44d)

**Comment by 07510480632** [comment]
84131

**Comment by butterfly0625** [comment]
Thank you so much to everyone who wanted to help out!

**Comment by Pelumijo** [comment]
> hi

Thanks

**Comment by mishmanners** [comment]
Love this! It's a fantastic graphic and congrats to the community for being so active.

**Reply by lovelyMircho** [reply]
Fuck u

**Comment by srajan07** [comment]
great to see

**Comment by mahdimohamadpoor** [comment]
thanks a lot

**Reply by apusaiiafahmad12** [reply]
https://github.com/d4rken-org/permission-pilot/commit/1644f21b107dbbae50f5a4a51fe4e3d44a144691

**Reply by RohRakun77** [reply]
Gm

**Comment by Muhammad-Rebaal** [comment]
Nice !

**Comment by vertias782** [comment]
good for learn

**Comment by Wan10** [comment]
Hello everybody 🤘

**Comment by HOMAYOUN1345** [comment]
Hi

**Comment by yehiamitro111** [comment]
Thankyou

**Comment by Soeparmadi** [comment]
Good

Pada tanggal Jum, 26 Jan 2024 22.14, butterfly0625 ***@***.***>
menulis:

**Comment by Jaxs88** [comment]
Thank u

**Comment by gemy6315** [comment]
> 2023 was an epic year for our Community, boasting nearly **double** the volume and total solutions authored as last year...all thanks to you! You make it possible for us to continue building around the launch of new & innovative products and improved user experience & accessibility!
> 
> Take a minute to be proud of your contribution and celebrate the start of the new year by checking out some highlights from 2023, and if you want, take an opportunity to share gratitude with members who had a positive impact on you in the Community. Add a comment below to say thank you! 🥳 
> 
> <img src="https://github.com/community/community/assets/144172738/2b8d9561-2ea0-47c7-83cf-41d7cfc0a83f" alt="image" width="50%" height="auto">
> 
> 
> ## Alt Text: Infographic entitled GitHub Community - 2023 Year in Review
> 
> ### Long Description:
> The infographic provides a snapshot of different data points from the GitHub Community over the year 2023. It has a purple, abstract background that simulates the shapes of a lava lamp. 
> 
> There are five different graphics to correspond with five different data points and descriptions. The graphics and descriptions are as follows: 
> 
> A cartoon graphic of a person typing on their laptop that is pink, purple and beige, next to a header entitled “23,546 Total Discussions”, and the text "The Community was extra busy this year, almost doubling the volume of discussions we saw in 2022!".
> 
> Below, a graphic of several people putting together puzzle pieces that form a lightbulb that is beige, purple, and blue with the header: "9,824 Solutions Offered", and the text: "Our members continued to provide the value of community by dishing out over 9,000 answers throughout the year!"
> 
> Then, an orange, purple and blue graphic of three people exchanging messages in a circle is paired with the header "Most Active
> Categories", followed by the list: 1. Actions - 3,682 posts, 2. Education - 2,583 posts and 3. Copilot - 2,150 posts
> 
> Next, an orange and pink graphic of a message box with a heart inside is next to the header: "Biggest Month" and the text: "The Community was on fire in August, creating a total of 2,253 posts. The average posts/per month for 2023 was 1,962."
> 
> Lastly, there is multicolored graphic of 5 people joined by a thread with the header: "Most Active Members" and text: "These members went above and beyond to be outstanding community citizens: @jge162, @airtower-l una, @jsoref, @GhostOf0days, @linuxlawson
> 
> ### Alt Title: Infographic for GitHub Community in 2023
> 
> ### File Name: GitHub Community Wrapped 2023.png
> 
> 

[]()@[]()

**Comment by srajan07** [comment]
can anyone tell me to how to participate in gsoc

**Comment by cryptofino** [comment]
hello !
wish best for everyone

**Comment by charlestanev** [comment]
I'm thrilled to be part of this amazing community and to hear about the incredible achievements in 2023! It's inspiring to see the dedication and contributions of each member, and I'm eager to celebrate your successes. Let's continue to support and uplift each other as we embark on another year of growth and collaboration.

Best wishes,

**Comment by charlestanev** [comment]
I'm new here and just wanted to express my appreciation for the vibrant and active community we have. The 2023 Year in Review highlights are truly impressive, and it's clear that everyone's hard work and dedication have made it possible. Looking forward to being part of more exciting discussions and milestones together!

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
